db/re/item_db.txt

```diff
@@ -11642,7 +11642,7 @@
 23228,Hazy_Mooncake,Hazy Mooncake,0,768,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ /* unknown */ },{},{}
 //
 23256,Elixir_Bandages,Elixir Bandages,2,10,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ pet 1041; },{},{}
-23277,Mado_Box,Emergency Magic Gear,2,10000,,3000,,,,,0x00000400,56,2,,,100,,,{ setmadogear 1; },{},{}
+23277,Mado_Box,Emergency Magic Gear,2,10000,,3000,,,,,0x00000400,56,2,,,100,,,{ specialeffect2 EF_CLAYMORE; setmadogear 1,MADO_ROBOT; },{},{}
```

Comment:
```suggestion
23277,Mado_Box,Emergency Magic Gear,2,10000,,3000,,,,,0x00000400,56,2,,,100,,,{ specialeffect2 EF_CLAYMORE; setmadogear true,MADO_ROBOT; },{},{}
```
Should we really put the mado type here, since it is default anyway? I dont mind to make it clear, just asking.